src/components/dashboard/tests/DashboardPage.test.jsx

```diff
@@ -455,14 +500,16 @@ describe('<Dashboard />', () => {
       );
       expect(screen.queryByText(SUBSCRIPTION_EXPIRING_MODAL_TITLE)).toBeTruthy();
       expect(screen.queryByText(SUBSCRIPTION_EXPIRED_MODAL_TITLE)).toBeFalsy();
+      expect(screen.getByText(expectedTimeDiff)).toBeTruthy();
+      expect(screen.getByText(expectedText)).toBeTruthy();
     });
 
-    it('should render the expired version of the modal when 0 >= daysUntilExpiration', () => {
+    it('should render the expired version of the modal when 0 >= daysUntilExpirationIncludingRenewals', () => {
       useSubscriptions.mockReturnValue({
         data: {
           showExpirationNotifications: true,
           subscriptionPlan: {
-            daysUntilExpiration: 0,
+            daysUntilExpirationIncludingRenewalsIncludingRenewals: 0,
```

Comment:
nit: typo and perhaps include `isCurrent: false`?

```suggestion
            daysUntilExpirationIncludingRenewals: 0,
            isCurrent: false,
```